packages/scenario/package.json

```diff
@@ -54,5 +54,8 @@
     "solid-js": "^1.9.3",
     "vite": "^5.4.11",
     "vitest": "^2.1.8"
+  },
+  "publishConfig": {
```

Comment:
Double checking:

Why do we publish scenario? 
Should we publish common? As it has things that other packages might need - perhaps is already bundled when publishing then please ignore the question

Reply:
😬 we don't publish scenario. I've removed that change.

I believe that the Vue client does end up bundling common so that ends up working. But I think currently `xforms-engine` doesn't work on its own. Let's consider that a separate work item.